### PR TITLE
Improvements for Pulseaudio package (conflicts, autorelease, simplifying)

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: @tripolar
Compile tested: Raspberry PI 4 CM, aarch64_cortex-a72_musl, OpenWrt snapshot
Run tested: not yet

Description:
- Many changes, see the commit descriptions

### CODE REVIEW

**Comment by xabolcs** [comment]
:pray:

File: sound/pulseaudio/Makefile [hunk 1]
```diff
@@ -157,140 +148,62 @@ endif
...
-		$(1)/usr/lib/
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pulseaudio/* \
-		$(1)/usr/lib/pulseaudio/
+	$(INSTALL_DIR) $(1)/usr/include/pulse
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/pulse/* $(1)/usr/include/pulse
+
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/*.pc $(1)/usr/lib/pkgconfig
+	$(SED) 's,/usr/include,$$$${prefix}/include,g' $(1)/usr/lib/pkgconfig/libpulse.pc
```
Comment (neheb) [inline sound/pulseaudio/Makefile:157]: Something tells me this is outdated…

Comment (BKPepe) [reply sound/pulseaudio/Makefile:157]: I agree with you, but I would rather not touch or modify it in this PR since it was there and avoid doing more changes than necessary right now. I need to rework commit messages, hopefully, I will do that tomorrow and then after merging this, we can look at it further. Those three lines things were really hard to follow and compare.

**Comment by hnyman** [comment]
cc @BKPepe 

The change regarding conffiles by 8b6db4b looks wrong to me.

It is now

https://github.com/openwrt/packages/blob/master/sound/pulseaudio/Makefile#L62-L69

```
define Package/pulseaudio-daemon/conffiles
/etc/pulse/client.conf
/etc/pulse/daemon.conf
/etc/pulse/default.pa
/etc/pulse/system.pa
endef

Package/pulseaudio-daemon/conffiles = $(Package/pulseaudio-daemon-avahi/conffiles)
```

which first defines `pulseaudio-daemon/conffiles` section and then one line later re-defines it as an assignment from a non-existing `pulseaudio-daemon-avahi/conffiles` section

Apparently the wrong section was deleted by 8b6db4b

Either there should be 
* a full `pulseaudio-daemon-avahi/conffiles` section and then the current assignment, or
* the current  `pulseaudio-daemon/conffiles` section and then the reversed assignment to `pulseaudio-daemon-avahi/conffiles` from that

**Comment by BKPepe** [comment]
Oh yeah. 🙈 I will fix that. Thanks for pointing it out

Dne út 1. 11. 2022 18:06 uživatel Hannu Nyman ***@***.***>
napsal:
